src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const listWithEmployees = document.querySelector('ul');
+const listWithEveryEmployee = document.querySelectorAll('li');
```

Comment:
названия путают, первый лист в смысле элемент ul, второй лист в смысле массив
```suggestion
const ulEmployees = document.querySelector('ul');
const employees = document.querySelectorAll('li');
```